python/pyspark/sql/types.py

```diff
@@ -1521,6 +1521,19 @@ def toPython(self) -> Any:
         """
         return VariantUtils.to_python(self.value, self.metadata)
 
+    def toJson(self, zone_id: str = "UTC") -> str:
```

Comment:
You might need to update this https://github.com/databricks/runtime/blob/master/python/docs/source/reference/pyspark.sql/variant_val.rst?plain=1#L27
to generate the docs.

Reply:
Thanks! I was trying to figure out why document generation was failing in the linting jobs. I thought it was a syntax error in Sphinx

Reply:
Resolved